.eslintrc

```diff
@@ -7,7 +7,32 @@
   "ignorePatterns": [
     "/fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/A32NX/**",
     "/fbw-common/src/typings/**",
-    "/fbw-common/msfs-avionics-mirror/**"
+    "/fbw-common/msfs-avionics-mirror/**",
+    "build*.js",
+    "buildSrc/*",
+    "igniter.config.mjs",
+    "fbw-common/msfs-avionics-mirror/*",
+    "fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/math.js",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/DirectTo.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/FixNamingScheme.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanAsoboSync.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanManager.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/FlightPlanSegment.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/GeoMath.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/GPS.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/LegsProcedure.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/ManagedFlightPlan.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/ProcedureDetails.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/RawDataMapper.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/SegmentedFlightPlan.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/WaypointBuilder.ts",
+    "fbw-a32nx/src/systems/fmgc/src/flightplanning/WorldMagneticModel.ts",
+    "fbw-a32nx/src/systems/fmgc/src/utils/LzUtf8.js",
+    "target/*",
+    "CMakeFiles/*",
+    "cmake*",
+    "fbw-a32nx/out/*",
+    "fbw-a380x/out/*"
   ],
   // DO NOT CHANGE THIS FILE (unless you are an admin) (if admin, make sure you change the same file in the latest asobo branch as well)
```

Comment:
This can be removed or changed I guess :)

Reply:
I don't think we want linting on our generated files

Reply:
I meant the comment `// DO NOT....`